src/components/workspace-utils.js

```diff
@@ -190,6 +197,7 @@ export const SnapshotInfo = ({
             key: id,
             style: { marginBottom: '1rem' }
           }, [
+            h(SnapshotLabeledInfo, { title: 'Source Data Repo Id:', text: [id, h(ClipboardButton, { 'aria-label': 'Copy source data repo id to clipboard', text: id, style: { marginLeft: '0.25rem' } })] }),
```

Comment:
could this be a link too? it would go to /datasets/${id}, same way the snapshot goes to /snapshots/${snapshotId}.

Reply:
Also, the terminology "Source Data Repo Id" here could be confusing. I'd suggest either "Data Repo Id" to use the same terminology as the previous link, or call the snapshot "Data Repo Snapshot Id" and this one "Data Repo Dataset Id" - but those should be Alessandro's call!